### ISSUE TITLE
`no-duplicate-id` rule false positive with block param

### ISSUE THREAD

**Issue by bmish** [ember-template-lint/ember-template-lint#1859]
The rule sees `id={{inputProperties.id}}` on multiple elements and flags a violation, but this is fine because the variable is scoped to the parent component.

```hbs
<MyComponent as |inputProperties|>
    <Input id={{inputProperties.id}} />
</MyComponent>

<MyComponent as |inputProperties|>
    <Input id={{inputProperties.id}} />
</MyComponent>
```

**Comment by faith-or** [comment]
I have a potential fix: https://github.com/ember-template-lint/ember-template-lint/pull/2024

**Comment by bertdeblock** [comment]
Can be closed right?

**Comment by faith-or** [comment]
Yup @bmish / @scalvert  can you close this since the fix is merged already
